src/cloud/iotile-cloud-serv.ts

```diff
@@ -588,6 +586,30 @@ export class IOTileCloud {
     });
   }
 
+  public async deleteStreamData(dataId: number) {
```

Comment:
I wonder if we should instead have a more generic `delete(resource: string, id: number)` where resource is either `note` or `data` or anything else that supports delete